src/index.html

```diff
@@ -10,7 +10,51 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form
+      action="big"
+      method="get"
+      class="form search__big"
+      data-qa="big"
+    >
+
+      <img
+        src="./images/Search.svg"
+        alt="magnifying glass"
+        class="icon icon_big"
+      >
+
+      <input
+        type="text"
+        class="input input__big"
+        name="query"
+        placeholder="Try &quot;Los Angeles&quot;"
+        autocomplete="on"
+        data-qa="keypress"
+      >
+    </form>
+
+    <form action="small"
+      method="get"
+      class="form search__small"
+      data-qa="small"
+    >
+
+      <img
+          src="./images/Search.svg"
+          alt="magnifying glass"
+          class="icon icon_small"
+      >
```

Comment:
```suggestion
      <img
        src="./images/Search.svg"
        alt="magnifying glass"
        class="icon icon_small"
      >
```
Two spaces are enough